File: src/main/java/guru/xgm/angelscript/psi/AngelScriptTokenSets.java
package guru.xgm.angelscript.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.angelscript.psi.AngelScriptTypes.*;

public interface AngelScriptTokenSets {
    TokenSet COMMENTS = TokenSet.create(SINGLE_LINE_COMMENT, MULTI_LINE_COMMENT);
}

File: src/main/kotlin/raft/war/language/angelscript/psi/AngelScriptTokenSets.kt
package raft.war.language.angelscript.psi

import com.intellij.psi.tree.TokenSet
import raft.war.language.angelscript.psi.AngelScriptTypes.*

interface AngelScriptTokenSets {
    companion object {
        val COMMENTS: TokenSet = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT)
    }
}
